src/js/post-911-gib-status/actions/post-911-gib-status.js

```diff
@@ -35,20 +35,25 @@ export function getEnrollmentData() {
         });
       },
       (response) => {
-        if (response.status === 503 || response.status === 504) {
-          // Either EVSS or a partner service is down or EVSS times out
-          return dispatch({ type: BACKEND_SERVICE_ERROR });
-        }
-        if (response.status === 403) {
-          // Backend authentication problem
-          return dispatch({ type: BACKEND_AUTHENTICATION_ERROR });
-        }
-        if (response.status === 404) {
-          // EVSS partner service has no record of this user
-          return dispatch({ type: NO_CHAPTER33_RECORD_AVAILABLE });
+        const error = response.errors.length > 0 ? response.errors[0] : undefined;
+        if (error) {
+          if (error.status === '503' || error.status === '504') {
+            // Either EVSS or a partner service is down or EVSS times out
+            return dispatch({ type: BACKEND_SERVICE_ERROR });
+          }
+          if (error.status === '403') {
+            // Backend authentication problem
+            return dispatch({ type: BACKEND_AUTHENTICATION_ERROR });
+          }
+          if (error.status === '404') {
+            // EVSS partner service has no record of this user
+            return dispatch({ type: NO_CHAPTER33_RECORD_AVAILABLE });
+          }
+          return Promise.reject(
+            new Error('post-911-gib-status getEnrollmentData() received unexpected error: `${error.status}: ${error.title}: ${error.detail}`'));
         }
         return Promise.reject(
-          new Error('post-911-gib-status getEnrollmentData() received unexpected status code: `${response.status}`'));
+          new Error('post-911-gib-status getEnrollmentData() received unexpected error'));
```

Comment:
At this point, there is no error information available (errors array was empty)--I'm not sure whether this could happen. I can add a 'no status code available' message to the explanation though.